src/components/Filters/Filters.tsx

```diff
@@ -1,115 +0,0 @@
-import React, { useState } from 'react';
-import styles from './Filters.module.css';
-import Button from '../Button';
-import { FiltersCheckboxField } from '../FiltersCheckboxField/FiltersCheckboxField';
-import { FiltersDrawer } from '../FiltersDrawer/FiltersDrawer';
-import Icon from '../Icon';
-
-export type Props = {
-  /**
-   * Text to be placed in the button that activates the Filters Drawer
-   */
-  triggerText?: string;
-  /**
-   * Indicates status that filters have been selected, influencing toggle button variant.
-   */
-  hasSelectedFilters?: boolean;
-  /**
-   * Input components, input component fields, or relevant information that will be displayed in the filters drawer.
-   */
-  children: React.ReactNode;
-  /**
-   * CSS class names that can be appended to the component.
-   */
-  className?: string;
-  /**
-   * Callback called when the clear button is called.
-   */
-  onClear?: () => void;
-  /**
-   * CSS class names that can be appended to the footer button group.
-   */
-  footerButtonGroupClassName?: string;
-  /**
-   * Callback called when filters drawer is closed.
-   */
-  onClose?: () => void;
-  /**
-   * Callback called when the apply button is called.
-   */
-  onApply?: () => void;
-};
-
-/**
- * BETA: This component is still a work in progress and is subject to change.
- *
- * ```ts
- * import {Filters} from "@chanzuckerberg/eds";
- * ```'
- *
- * A filter component with a button that triggers a drawer of checkbox filters to be selected.
- */
-export const Filters = ({
```

Comment:
Are filters only going to be available in the context of a drawer or popover? Do we see no reason to expose the filter itself so it could be placed on a page where it's always visible? Just curious.

Reply:
That's a good question, and I'll bring it up to design. However I'm thinking if product teams need to have filters available on the page, wouldn't it just be the `FiltersCheckboxField` component, other `<fieldsets>`, or simply other input elements that would be on the page rather than being wrapped by `FiltersPopover` or `FiltersDrawer`?

Reply:
That's fair 👍